### PR TITLE
SNS topic class with publish method

### PR DESCRIPTION
**Pull request by DomBelcher**
The webhook handler lambda should write data from the webhook events to SNS topics.

This provides a helper class to wrap the AWS SDK method to publish to an SNS topic, as well as formatting the data to be published as stringified JSON.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,8 +20,12 @@
   },
   "homepage": "https://github.com/lulibrary/LAG-Utils#readme",
   "devDependencies": {
```
Comment (stephenrob) [inline package.json:22]: There's no Dependency for the AWS-SDK that you require in the Topic class, can you add this as one.

File: src/topic.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+class Topic {
+  constructor(arn, region) {    
+    this.arn = arn;
+    this.sns = new AWS.SNS({ region });
+  }
+
+  publish (data) {
+    let params = this.generateSnsMessage(data);
+    // console.log(params)
```
Comment (stephenrob) [inline src/topic.js]: Do we require a `console.log` here? If this is just for debug would be worth removing.

**Review by stephenrob — CHANGES_REQUESTED** [review]
Changes to ensure AWS-SDK dependency is required.

File: package.json (hunk 1 above)
Comment (DomBelcher) [reply package.json:22]: This is  fixed in https://github.com/lulibrary/LAG-Utils/pull/2/commits/502d786450479aefef2453f7086aa3459088afd1

File: src/topic.js (hunk 2 above)
Comment (DomBelcher) [reply src/topic.js]: This is fixed in https://github.com/lulibrary/LAG-Utils/pull/2/commits/502d786450479aefef2453f7086aa3459088afd1

**Review by stephenrob — APPROVED** [review]

**Pull request merged**